src/gui/FileBrowser.cpp

```diff
@@ -567,10 +567,17 @@ void FileBrowserTreeWidget::keyPressEvent(QKeyEvent * ke )
 	else if (vertical || horizontal || preview || insert) { stopPreview(); }
 
 	// Try to get the currently selected item as a FileItem
-	auto file = dynamic_cast<FileItem*>(currentItem());
-	// If it's null (folder, separator, etc.), there's nothing left for us to do
-	if (file == nullptr) { return; }
+	QTreeWidgetItem* item = currentItem();
 
+	auto file = dynamic_cast<FileItem*>(item);
+
+	// If it's not a file, try to expand it.
+	if (file == nullptr) {
```

Comment:
Looks like any key will expand it, not just space. Haven't checked tho.

Reply:
no, it checks if preview is true. preview is only true if space is pressed.

Reply:
Ok then